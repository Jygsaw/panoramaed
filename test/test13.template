<html>
  <head>
    <title><* page.title *></title>
  </head>
  <body>
    <h1><* teacher.name *></h1>
    Text
  </body>
</html>
