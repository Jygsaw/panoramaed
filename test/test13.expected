<html>
  <head>
    <title>Templating Example</title>
  </head>
  <body>
    <h1>Mr. Panorama</h1>
    Text
  </body>
</html>
